app/assets/stylesheets/components/_cards.scss

```diff
@@ -0,0 +1,33 @@
+.card-category {
+
+  width:300px;
+  align-items: center;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid $gold;
+  border-radius: 4px;
+  transition: .3s;
+
+
+  .category-image {
+    width: 300px;
+    height: 300px;
+    object-fit: cover;
+  }
+
+  .text-category-card {
+    text-decoration: none;
+    color: $gold;
+  }
+
+  h3 {
+    padding: 16px;
+    text-align: center;
+    text-shadow: 2px 2px 4px $main-grey;
+  }
+}
+
+.card-category:hover {
+  background-color: $gold;
+}
```

Comment:
A card component needs to be its own PR so you need to remove that from this one.
Rutger is currently working on one, you can share your ideas and work on the card component together if you want.

For now the only thing that you need in the homepage is a container for the cards.
Then later we just need to inject the cards in that existing container and it'll be easy.